components/icon-button/index.js

```diff
@@ -10,11 +10,11 @@ import './style.scss';
 import Button from '../button';
 import Dashicon from '../dashicon';
 
-function IconButton( { icon, children, label, className, ...additionalProps } ) {
+function IconButton( { icon, children, label, className, focus, ...additionalProps } ) {
```

Comment:
`focus` would be passed to the underlying `Button` without being explicitly handled here, based on the behavior of `additionalProps` spread.